content/news/2015/10/2015-10-07-the-data-briefing-white-house-asks-data-scientists-and-app-developers-to-help-suicide-prevention-efforts.md

```diff
@@ -6,7 +6,6 @@ summary: 'The White House issued a call on September 30, 2015, for data scientis
 authors:
   - bbrantley
 topics:
-  - data
   - mobile-apps
```

Comment:
```suggestion
  - mobile-apps
  - software-engineering
```

Reply:
Remove `the-data-briefing` and `native-apps`. Replace `mobile-apps` with `mobile`

Also left comment in 60-day spreadsheet about the `the-data-briefing` topic.